scripts/components/dropdown.component.js

```diff
@@ -9,7 +9,7 @@ export default class {
     this.title = this.el.querySelector('.js-dropdown-title');
```

Comment:
This particular component is used everywhere. I found out the hard way that changes to it can spread caos. please be sure to test this carefully and on all places it's currently used